docs/content/preview/explore/ycql-language/keyspaces-tables.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Keyspaces and tables
+linkTitle: Keyspaces and tables
+description: Keyspaces and tables in YCQL
+image: /images/section_icons/secure/create-roles.png
+menu:
+  preview:
+    identifier: explore-ycql-language-features-keyspaces-tables
+    parent: explore-ycql-language
+    weight: 60
+type: docs
+---
+
+This page explores keyspaces and tables in YCQL using the command line shell `ycqlsh`.
+
+{{% explore-setup-single %}}
+
+## YCQL shell
+
+Use the [ycqlsh shell](../../../admin/ycqlsh/) to interact with a Yugabyte database cluster using the [YCQL API](../../../api/ycql/). Because `ycqlsh` is derived from the Apache Cassandra shell [cqlsh](https://cassandra.apache.org/doc/latest/cassandra/tools/cqlsh.html), most `cqlsh` commands work as is in `ycqlsh`. Unsupported commands are [listed](#unsupported-cqlsh-commands) at the end of this page.
+
+Using `ycqlsh`, you can:
+
+- interactively enter YCQL DDL and DML and see the query results
+- input from a file or the command line
+- use tab completion to automatically complete commands
+- use a minimal command name as long as it can be distinguished from other commands (for example, `desc` instead of `DESCRIBE`)
+
+`ycqlsh` is installed with YugabyteDB and is located in the `bin` directory of the YugabyteDB home directory.
+
+### Connect to a node
+
+From your YugabyteDB home directory, connect to any node of the database cluster as follows:
+
+```sh
+$ ./bin/ycqlsh 127.0.0.1
+```
+
+This should bring up the following prompt, which prints the version of `ycqlsh` being used.
+
+```output
+Connected to local cluster at 127.0.0.1:9042.
+[ycqlsh 5.0.1 | Cassandra 3.9-SNAPSHOT | CQL spec 3.4.2 | Native protocol v4]
+Use HELP for help.
+ycqlsh>
+```
+
+## Users
+
+By default, YugabyteDB has one admin user already created: `cassandra` (the recommended user), and you can check the existing user as follows:
+
+```sql
+ycqlsh> select * from system_auth.roles;
+```
+
+You should see an output similar to the following:
+
+```output
+ role      | can_login | is_superuser | member_of | salted_hash
+-----------+-----------+--------------+-----------+------------------------------------------------------------------------------
+ cassandra |      True |         True |          [] | $2a$12$64A8Vo0R3K9XeUp26CSzpuWtvUBwOiGFjPAbXGt7wsxZIScGrcsDu\x00\x00\x00\x00
+```
+
+## Keyspaces
+
+YCQL supports keyspaces, much like Apache Cassandra, with one caveat. Keyspaces in Apache Cassandra
```

Comment:
- Could you explain the concept of Keyspaces tad more (like column families etc).

Reply:
Sure.

Reply:
Hows this?

> A keyspace is a container for a set of tables that belong together somehow. If multiple applications are using a single database, the tables belonging to each application can be grouped into a separate keyspace. In Apache Cassandra, replication configuration is done at the keyspace level. So, all tables that require the same replication configuration may be grouped into a keyspace. This pattern does not apply to YugabyteDB because in YugabyteDB replication configuration is done at the [universe](../../../architecture/concepts/universe/) level.

Reply:
Looks good !!